ephios/plugins/basesignup/templates/basesignup/fragment_state_common.html

```diff
@@ -4,14 +4,18 @@
     {% for participation in participations %}
         {% include "basesignup/participation_card_inline.html" %}
     {% endfor %}
-    {% for i in empty_spots %}
+    {% for index in empty_spots %}
         <span class="badge bg-light px-5">&nbsp;</span>
     {% endfor %}
 </div>
 
-<span data-bs-toggle="tooltip" data-bs-placement="top"
-      title="{% translate 'Needed' %}: {{ shift.signup_method.get_participation_number_info }}"
-><strong>{{ confirmed_count }}</strong> {% translate "in total" %}</span>
+{% with stats=shift.signup_method.get_signup_stats %}
+    <span class="pe-1">
+        <strong>{{ stats.confirmed_count }}{% if stats.max_count %}/{{ stats.max_count }}{% endif %}</strong>
+        {% translate "Total" context "participation count" %}
```

Comment:
I started making use of context in the translation system, here in the template and in python using `pgettext`. Make sure you know what that is ([Docs](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#contextual-markers)) and respect it when translating.

Reply:
i know that, will be interesting to see if weblate knows it :P